### DISCUSSION TITLE
[Bug?] Unable to access settings page for Collections

### DISCUSSION THREAD

**Discussion by SimonXIX** [knowledgefutures/pubpub#2632] [Support]
Hi,

I'm currently unable to access the Settings page for any Collections. I've tried this on our https://ecologicalrewritings.pubpub.org/ and our https://copim.pubpub.org/ sites. The page attempts to load, hangs for about three minutes, and then fails with an error in the console reading 'Failed to load resource: net::ERR_HTTP2_PROTOCOL_ERROR'. 

The specific URL for the Collection I'm trying to edit is https://ecologicalrewritings.pubpub.org/dash/collection/full-text/settings. 

Thanks,

Simon

**Comment by gabestein** [comment]
Sorry about that Simon. Should be fixed shortly.
